pkg/wamr/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
Not very experiences in `cmake` to review this, maybe @cgundogan or @leandrolanzieri can take a look?